include/kwk/settings/axis.hpp

```diff
@@ -103,7 +103,13 @@ namespace kwk::__
       return axis_<ID,decltype(v)>{v};
     }
 
-    Content value = {};
+    KWK_PURE explicit constexpr operator Content          () const noexcept { return value; }
+    KWK_PURE          constexpr          Content operator*() const noexcept { return value; }
+
+#ifndef _WIN32 // https://github.com/llvm/llvm-project/issues/49358 Missing [[no_unique_address]] support on Windows
```

Comment:
can this be put in a macro in detail/abi.hpp ?
Also is it really necessary as axis are mainly used as constexpr entity ?

Reply:
i want the axis thingy to behave as a 'unit' wrapper (like boost.units)...wait for other changes..